conceptnet5/readers/emoji.py

```diff
@@ -0,0 +1,51 @@
+from conceptnet5.edges import make_edge
+from conceptnet5.formats.msgpack_stream import MsgpackStreamWriter
+from conceptnet5.uri import Licenses
+from conceptnet5.nodes import standardized_concept_uri
+
+import xml.etree.ElementTree as ET 
+
+
+REL = '/r/SymbolOf'
+DATASET = '/d/emoji'
+LICENSE = Licenses.cc_attribution
+SOURCE = [{'contributor': '/s/resource/unicode/cldr/31'}]
+
+
+def is_sentence(text):
+    """
+    There are a few instances where a sentence of
+    multiple words is used to describe an emoji
+    (which is not very helpful in our case) AS WELL AS
+    single words or phrases, which are separated by '|'.
+    Using this function, we can ignore the sentences/phrases,
+    and only look at single words, which is better for
+    conceptnet to handle.
+    """
+    return (' ' in text and '|' not in text)
```

Comment:
I'm still not sure what this is meant to filter out. Is it the phrases in the entries marked "tts", like "money with wings"? I don't see a need to remove those. Some of them, such as "fortune cookie", are valuable phrases to have mappings for.